java-service.gitlab-ci.yml

```diff
@@ -27,7 +27,7 @@ before_script:
   - export PACKAGE_VERSION=${CI_COMMIT_SHA:0:7}
 
 maven_build:
-  image: maven:3.5.0-jdk-8-alpine
+  image: maven:3.6.1-jdk-8-alpine
```

Comment:
You could just go 3.6 instead of 3.6.1 - not a big deal either way. This is a good change since it gets us to a modern JDK version (3.5 was pegged to 131, which came out in 2017!)

Reply:
yeah, pretty safe to use 3.6. even 3.x.x. I can update to 3.6